index.html

```diff
@@ -60,23 +61,53 @@ <h1 class="popuplabel">Custom Colors</h1>
 </div>
   <div class="popup" id="moreStuff">
     <div class="x" onclick="closeStuffPanel()">&#215;</div>
+    <h1 class="popupheader">Weather</h1>
```

Comment:
change to `Weather at School` so that they know it's not their current location but at parkland